src/library_glfw.js

```diff
@@ -382,6 +385,38 @@ var LibraryGLFW = {
 #endif
     },
 
+    onGamepadConnected: function(event) {
+      if (!GLFW.active.joystickFunc) return;
```

Comment:
I wonder if GLFW should be tracking gamepads already before the user code has registered a joystick callback function. For example, it can take a while until the user code runs, and a gamepad connected event may have fired already before. Or perhaps the tightest way would be to enumerate all already known joysticks when a callback is registered or GLW initialized to initially populate the already known joysticks. I'm thinking about the scenario where user is clicking on gamepad buttons way before the page has fully loaded up.